apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -96,4 +99,22 @@ export const pageRouter = router({
       return { pageId: "" }
     }),
   // TODO: Delete page stuff here
+
+  readImageInPage: pageProcedure
+    .input(readImageInPageSchema)
+    .query(async ({ input, ctx }) => {
+      // NOTE: If image is not publically accessible, might need to add logic to get it
+      const res = await fetch(input.imageUrlInSchema)
+      const blob = await res.blob()
+      const splitUrl = input.imageUrlInSchema.split(".")
+      const extension = splitUrl[splitUrl.length - 1] || ""
+      const base64Data = await BlobToImageDataURL(blob, extension)
+      return { imageDataURL: base64Data }
+    }),
+  uploadImageGetURL: pageProcedure
```

Comment:
commenting only since the image upload portion has not been completed yet. 
feels a little anti-patterny to upload images to the server. What if users uploads a 100mb image etc. 
have we decided how to handle image uploads? the usual flow is to get a presigned url from aws s3 that the user uploads directly to using the client. 

[starter kit has an example flow](https://github.com/opengovsg/starter-kit/blob/main/src/features/posts/api/useUploadImagesMutation.ts) you may want to check out.